ntbs-service/Models/Entities/CultureAndResistance.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using ntbs_service.Helpers;
+
+namespace ntbs_service.Models.Entities
+{
+    public class CultureAndResistance

```

Comment:
I think this needs a `[NotMapped]` clause so EF doesn't try to pick it up in migrations